spec/controllers/users_controller_spec.rb

```diff
@@ -77,6 +77,13 @@
       expect(user.username).to eq("user234")
       expect(response).to redirect_to facility_user_path(facility, user)
     end
+
+    it_should_allow(:admin, "to log the change of the internal flag and log the new rate as metadata") do
+      log_event = LogEvent.find_by(loggable: user, event_type: :internal_changed)
+      expect(log_event).to be_present
+      expect(log_event.metadata).to eq("price_group_rate"=>"Northwestern Base Rate")
```

Comment:
Can we include spaces in the hash? 

```suggestion
      expect(log_event.metadata).to eq("price_group_rate" => "Northwestern Base Rate")
```